test/component/CustomsDeclarationSubmissionSpec.scala

```diff
@@ -97,6 +97,7 @@ class CustomsDeclarationSubmissionSpec extends ComponentTestSpec with AuditServi
       status(result) shouldBe ACCEPTED
```

Comment:
I think for the 3 services, in the component tests we need a call to `MdgWcoDecService.verifyMdgWcoDecServiceWasCalledWith` to verify the payload contains the new fields